tests/ci/benchmark_framework/compare_results.py

```diff
@@ -57,16 +57,18 @@ def main():
     # 2nd column of the dataframe represents the number of calls
     df1_numCalls = df1['numCalls.1']
     df2_numCalls = df2['numCalls.2']
+    df1_time = df1['microseconds.1']
+    df2_time = df2['microseconds.2']
 
     # put both dataframes side by side for comparison
     dfs = pd.concat([df1, df2], axis=1)
 
     # we want things that have a +15% regression
-    compared = np.where(((df2_numCalls - df1_numCalls)/df1_numCalls*100 <= -15), df1.iloc[:, 0], np.nan)
+    compared = np.where( 100 *( 1 - df2_time*df1_numCalls/df2_numCalls/df1_time) >= 15, df1.iloc[:, 0], np.nan)
```

Comment:
can you please write this with parentheses because it's not obvious to me what we are trying to compute (for example, I don't know how a/b/c computes in python :)).